packages/cli/test/unit/cmds/beacon.test.ts

```diff
@@ -60,7 +60,7 @@ describe("cmds / beacon / args handler", () => {
     fs.writeFileSync(peerIdFile, JSON.stringify(prevPeerId.toJSON()));
 
     const {peerId} = await runBeaconHandlerInit({
-      peerIdFile,
+      // peerIdFile,
```

Comment:
This test is skipped, not sure why Typescript didn't complain about this one before